keyboards/cxt12e4/keymaps/default/keymap.c

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 Ismaïl Senhaji (@iSma)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(/* Base */
+                 KC_BSPC, KC_7, KC_8, KC_9,
+                 KC_TAB,  KC_4, KC_5, KC_6,
+                 KC_0,    KC_1, KC_2, KC_3,
+                 RGB_MOD, RGB_TOG, KC_MPLY, KC_MUTE
+                 ),
+};
+
+#if defined(ENCODER_MAP_ENABLE)
```

Comment:
Could you add this to a `cxt12e4.c` file?  

This would allow for a default behavior, with the ability to be overridden by user config.  This is needed by the QMK Configurator, as it doesn't support encoders, at this time. 

```c
#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) {
        if (clockwise) {
            tap_code_delay(KC_VOLU, 10);
        } else {
            tap_code_delay(KC_VOLD, 10);
        }
    }
    return true;
}
#endif
```